docs/operate.md

```diff
@@ -2,11 +2,48 @@
 
 Information about operating the hubs, debugging problems, and performing common actions.
 
+(operate:team-process)=
+## Team process for operating hubs
+
+### Each hub gets an issue
+
+When a new hub is created, [create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new) using the "New Hub" template.
+This issue is a "meta" issue that tracks the state of this hub over time.
+These issues serve both as a "Source of Truth" of information about the hub, as well as a place for conversation around operating that hub.
+
+### Add the new hub issue to the hubs project
+
+We use [a GitHub Project board](https://github.com/2i2c-org/pilot-hubs/projects/1) to keep track of the tasks associated with running the hubs at 2i2c.
+Some of these tasks may be one of the hub issues detailed above.
+Other tasks may be a specific development or improvement item.
+
+### Weekly task syncs
+
+Each week, the 2i2c Hub Operations team reviews the issues that are currently on the 2i2c board in order to divide and conquor.
```

Comment:
We should probably define what / when this is. How do we make this be as asynchronous as possible?

Reply:
Let's just say 'on Monday'? And we all do a pass on Monday.